gradle/libs.versions.toml

```diff
@@ -10,6 +10,7 @@ appcompat = "1.6.1"
 material = "1.11.0"
 activity = "1.8.0"
 constraintlayout = "2.1.4"
+mockitoKotlinVersion = "2.2.0"
```

Comment:
`Mockito` does not seem to be used and we are already using `mock`, so please remove this dependency.

Reply:
is removed now